src/naive/isoweek.rs

```diff
@@ -151,6 +153,7 @@ impl fmt::Debug for IsoWeek {
 
 #[cfg(test)]
 mod tests {
+    use super::IsoWeek;
```

Comment:
```suggestion
    #[cfg(feature = "rkyv-validation")]
    use super::IsoWeek;
```

To prevent the unused import warning.